docs/configuration/configuration.md

```diff
@@ -868,7 +866,9 @@ role: <string>
 # Optional filters to limit the discovery process to a subset of available
 # resources.
 # The available filters are listed in the upstream documentation:
-# https://docs.docker.com/engine/api/v1.40/#operation/ContainerList
+# Services: https://docs.docker.com/engine/api/v1.40/#operation/ServiceList
+#    Tasks: https://docs.docker.com/engine/api/v1.40/#operation/TaskList
+#    Nodes: https://docs.docker.com/engine/api/v1.40/#operation/NodeList
```

Comment:
Is the indentation intentional here?

Reply:
Yes, I thought it looked nicer when the links aligned properly. It's personal preference, I can undo the alignment if you like.

Reply:
I would undo it because it looks like tasks and nodes depend on the first link. Thanks.

Reply:
Sure, I have reverted the indentation change. Should I squash the commits or are you doing that anyway during merge?